client/src/main/scala/com.gu.contentapi.client/model/Queries.scala

```diff
@@ -76,23 +76,43 @@ case class ItemQuery(id: String, parameterHolder: Map[String, Parameter] = Map.e
   with FilterExtendedParameters[ItemQuery]
   with FilterSearchParameters[ItemQuery] {
 
-  def withParameters(parameterMap: Map[String, Parameter]) = copy(id, parameterMap)
+  def withParameters(parameterMap: Map[String, Parameter]) = copy(id, parameterMap, channelId)
+
+  def withChannelId(newChannel:String) = copy(id, parameterHolder, Some(newChannel))
+
+  def withoutChannelId() = copy(id, parameterHolder, None)
 
   def itemId(contentId: String): ItemQuery =
     copy(id = contentId)
 
-  override def pathSegment: String = id
+  override def pathSegment: String = channelId match {
+    case None => id
+    case Some(chl) => s"channel/$chl/item/$id"
+  }
 }
 
-case class SearchQuery(parameterHolder: Map[String, Parameter] = Map.empty)
+case class SearchQuery(parameterHolder: Map[String, Parameter] = Map.empty, channelId: Option[String] = None)
   extends PaginatedApiQuery[SearchResponse, Content] with SearchQueryBase[SearchQuery] {
 
   def setPaginationConsistentWith(response: SearchResponse): PaginatedApiQuery[SearchResponse, Content] =
     pageSize.setIfUndefined(response.pageSize).orderBy.setIfUndefined(response.orderBy)
 
-  def withParameters(parameterMap: Map[String, Parameter]): SearchQuery = copy(parameterMap)
+  def withParameters(parameterMap: Map[String, Parameter]): SearchQuery = copy(parameterMap, channelId)
+
+  /**
+    * Make this search on a CAPI channel rather than against web-only content
+    * For more information about channels, and the reason why your app should only be in one channel,
+    * content the Content API team
```

Comment:
`content the Content API team` -> `contact the Content API team` :)

Reply:
damn touch typing! 😁